apps/marketing/src/app/(marketing)/open/typefully.tsx

```diff
@@ -0,0 +1,47 @@
+'use client';
+
+import type { HTMLAttributes } from 'react';
+import { useEffect, useState } from 'react';
+
+import Image from 'next/image';
+import Link from 'next/link';
+
+import Twitter from '@documenso/assets/twitter-icon.png';
+import { cn } from '@documenso/ui/lib/utils';
+import { Button } from '@documenso/ui/primitives/button';
+
+export type TypefullyProps = HTMLAttributes<HTMLDivElement>;
+
+export const Typefully = ({ className, ...props }: TypefullyProps) => {
+  const [isSSR, setIsSSR] = useState(true);
+
+  useEffect(() => {
+    setIsSSR(false);
+  }, []);
```

Comment:
We have a mounted hook for this but I'm not sure as to why you would need it for the following code.